### PR TITLE
Fix API request when method is HEAD

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: store/deposit-dates.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ class DepositDates {
   loadDepositDatesCount = async () => {
     try {
       this.isExportDisabled = false
-      const { headers } = await apiRequest(this.datesUrl, { mehtod: 'HEAD' })
```
Comment (viktor-yakubiv) [inline store/deposit-dates.js]: Thank you for fixing this.

**Comment by Joozty** [comment]
It fails without that condition. The body is not processed as a blob but as a json which is wrong. You sure you tested it correctly?

**Comment by viktor-yakubiv** [comment]
I tested HEAD request to Google if I remember so I can be sure that Google does not append `Content-Type` to empty response. We should not do it too @valerapro. However, if it works so, we have no other way for now. Thank you for fixing this.

**Comment by Joozty** [comment]
HEAD should mirror GET response except for the body, shouldn't it? Anyway, I am not expert in this. So whatever Google does is probably correct. 😄

**Pull request merged**